apps/firelens-stability/collections/ecs-firelens-stability-tests-adot/golden-path/case-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This is a minimized test-case containing the golden path. The only change is task counts are set to 1 rather than 40.